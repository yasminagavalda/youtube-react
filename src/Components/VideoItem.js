import React from 'react';

const VideoItem = (props) => {
  return (
  	<div>
  		<div>{props.video.title}</div>
	  	<img src={props.video.image} alt={props.video.title} />
	  </div>

  )
}


export default VideoItem