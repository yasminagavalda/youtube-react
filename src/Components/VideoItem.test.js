import React from 'react'
import VideoItem from './VideoItem'
import { shallow } from 'enzyme'

describe('VideoItem', () => {
	it('renders correctly', () => {

		const video = {
			title: "Pulp - Common People",
			id: "yuTMWgOduFM",
			image: "https://i.ytimg.com/vi/yuTMWgOduFM/mqdefault.jpg"
		}
 
		const component = shallow(<VideoItem video={video}/>)
		expect(component).toMatchSnapshot()
	})
})