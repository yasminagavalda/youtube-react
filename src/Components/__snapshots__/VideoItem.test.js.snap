// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoItem renders correctly 1`] = `
<div>
  <div>
    Pulp - Common People
  </div>
  <img
    alt="Pulp - Common People"
    src="https://i.ytimg.com/vi/yuTMWgOduFM/mqdefault.jpg"
  />
</div>
`;
